src/classes/UTIL_Describe.cls

```diff
@@ -680,6 +680,32 @@ public class UTIL_Describe {
         }
     }
 
+    /*******************************************************************************************************
+     * @description Returns the ID of the custom Administrative Account record type
+     * @return The ID of the custom Administrative Account record type.
+     */
+    public static Id getCustomAdminAccRecTypeId() {
+        Id customAdminRecTypeId = Util_CustomSettingsFacade.getSettings().Administrative_Account_Record_Type__c;
+        if (customAdminRecTypeId == null 
+            || getRecTypesMapByDevName('Account').values().contains(customAdminRecTypeId) == false) {
+            customAdminRecTypeId = getAdminAccRecTypeID();
```

Comment:
```suggestion
            customAdminRecTypeId = UTIL_Describe.getAdminAccRecTypeID();
```
When accessing static methods within the same class, please prepend them with the class name.